docker/README_DEV.md

```diff
@@ -1,16 +1,16 @@
 # Eclipse RDF4J server and workbench
 
-Documentation for building and distributing the Server and Workbench image
+Documentation for building and distributing the docker image containing the Server and Workbench.
 
 ## Getting started
 
-Run `./run.sh` to build and start the server and workbench.
+Run `./run.sh` to build and start the server and workbench. This will invoke the `build.sh` script.
 
-## Full build
+### Full build (build.sh)
 Before building the docker image we need to get hold of the WAR files. These are built when 
 the maven project is assembled and packed in the sdk ZIP-file under `assembly/target`.
 
-The Dockerfile expects that the sdk ZIP-file is copied and renamed as `ignore/rdf4j.zip`.
+The Dockerfile expects that the SDK ZIP-file is copied and renamed as `ignore/rdf4j.zip`.
```

Comment:
It's not clear to me what this means. Does the build script take care of this itself somehow, or am I as a dev supposed to manually execute this copy operation?

Reply:
Build script does this.

Reply:
Slightly changed the wording so it's clear that the script takes care of it